backend/src/packages/users/user.repository.ts

```diff
@@ -0,0 +1,47 @@
+import { type IRepository } from '~/libs/interfaces/interfaces.js';
+import { UserEntity } from '~/packages/users/user.entity.js';
+import { type UserModel } from '~/packages/users/user.model.js';
+
+class UserRepository implements IRepository {
+  private userModel: typeof UserModel;
+
+  public constructor(userModel: typeof UserModel) {
+    this.userModel = userModel;
+  }
+
+  public find(): ReturnType<IRepository['find']> {
+    return Promise.resolve(null);
+  }
+
+  public async findAll(): Promise<UserEntity[]> {
+    const users = await this.userModel.query().execute();
```

Comment:
in service methods and in create method it named as items
```suggestion
    const items = await this.userModel.query().execute();
```

Reply:
can be both